src/NuGet.Core/NuGet.Protocol/README.md

```diff
@@ -1 +1,218 @@
-NuGet's implementation for interacting with feeds. Contains functionality for all feed types.
+# NuGet.Protocol
+
+NuGet.Protocol is a NuGet client SDK library that provides a set of APIs for interacting with NuGet feeds. It provides a way for developers to query NuGet feeds to discover packages and their dependencies, and also to download packages and their associated assets.
+
+## Getting started
```

Comment:
I think how-to installs aren't appropriate for a package readme. More often than not, people see the readme later.

Reply:
I'm guessing you forgot to remove tihs?